Tic-Tac-Toe/Cell.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace TicTacToe
+{
+    internal class Cell
+    {
+        public int Row { get; set; }
+
+        public int Column { get; set; }
+
+        private Cell(int row, int column)
```

Comment:
a private constructor is only done when we don't want to create that object.
But this usually works for static classes.
This is more likely a mistake